README.md

```diff
@@ -70,10 +78,126 @@ ELEVENLABS_API_KEY=YOUR_KEY
 
 ### Usage
 
-To run the LMS you can use this command from the root, alternatively you can run this on the Molly and Flow branches.
+#### LMS
+
+1. **Build All Packages:** Before running the LMS, make sure all other dependencies are built:
+
+```bash
+npm run build
+```
+
+2 **Run the LMS:**
+
+```bash
+npm run dev
```

Comment:
Running this from the root scope will start LMS.

Alternatively you can run `npm run dev --workspace @mollify/lms` from the root scope.